src/west/main.py

```diff
@@ -140,9 +140,19 @@ def append(*strings):
                     self.format_command(append, command, width)
                 append('')
 
+            # TODO we may want to be more aggressive about loading
+            # command modules by default: the current implementation
+            # prevents us from formatting one-line help here.
+            #
+            # Perhaps a commands.external_paranoid that if set, uses
+            # thunks, and otherwise just loads the modules and
+            # provides help for each command.
+            #
+            # This has its own wrinkle: we can't let a failed
+            # import break the built-in commands.
             if self.west_externals:
                 for path, specs in self.west_externals.items():
-                    append('commands from project {} at path "{}":'.
+                    append('commands from project at "{}":'.
```

Comment:
Why removing project name here, i.e. `{}` but still having both `specs[0].project.name` and `path`
in the formatting string below ?
```format(specs[0].project.name, path))```

Reply:
Thanks for catching that! Fixed